<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToeGame</class>
 <widget class="QWidget" name="TicTacToeGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn00">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1</y>
     <width>160</width>
     <height>136</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: 0;
border: 2px solid black;
border-left: 0;
border-top: 0;
background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn01">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>1</y>
     <width>160</width>
     <height>136</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: 0;
border: 2px solid black;
border-top: 0;
background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn02">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>1</y>
     <width>160</width>
     <height>136</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: 0;
border: 2px solid black;
border-right: 0;
border-top: 0;
background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>137</y>
     <width>160</width>
     <height>136</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: 0;
border: 2px solid black;
border-left: 0;
background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>137</y>
     <width>160</width>
     <height>136</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: 0;
border: 2px solid black;
background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn12">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>137</y>
     <width>160</width>
     <height>136</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: 0;
border: 2px solid black;
border-right: 0;
background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>273</y>
     <width>160</width>
     <height>136</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: 0;
border: 2px solid black;
border-left: 0;
border-bottom: 0;
background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn21">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>273</y>
     <width>160</width>
     <height>136</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: 0;
border: 2px solid black;
border-bottom: 0;
background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn22">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>273</y>
     <width>160</width>
     <height>136</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: 0;
border: 2px solid black;
border-right: 0;
border-bottom: 0;
background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
